extensions/jdbc/jdbc-h2/runtime/src/main/java/io/quarkus/jdbc/h2/runtime/H2AgroalConnectionConfigurer.java

```diff
@@ -8,6 +8,12 @@
 @JdbcDriver(DatabaseKind.H2)
 public class H2AgroalConnectionConfigurer implements AgroalConnectionConfigurer {
 
+    @Override
+    public void contributeJdbcProperties(AgroalDataSourceConfigurationSupplier dataSourceConfiguration) {
+        dataSourceConfiguration.connectionPoolConfiguration().connectionFactoryConfiguration().jdbcProperty("DB_CLOSE_DELAY",
```

Comment:
Would that work for non-memory H2 databases too?

Reply:
Good question. Want to give it a try? :] (hey, you had to see that one coming!)

Reply:
We could find a solution if it was an issue but we would have to push more information via the API.

Reply:
Do you know an easy way to see the JDBC URL used when the application starts? https://quarkus.io/guides/datasource#jdbc-configuration doesn't help

Reply:
Just tested, it doesn't seem to be an issue.

Reply:
BUT I see the following error if I define this property in my `application.properties`:

```java 
Caused by: org.h2.jdbc.JdbcSQLException: Duplicate property "DB_CLOSE_DELAY" [90066-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.engine.ConnectionInfo.readSettingsFromURL(ConnectionInfo.java:272)
```

Reply:
> Do you know an easy way to see the JDBC URL used when the application starts?

I don't know. But that's not exactly what you want to do if you want to test things. You would need to get access to the JDBC properties map.

`AgroalDatasource#getConfiguration()` should have that.